app/src/sharedTest/java/org/oppia/android/app/options/OptionsFragmentTest.kt

```diff
@@ -102,6 +102,7 @@ class OptionsFragmentTest {
     FirebaseApp.initializeApp(context)
   }
 
+
```

Comment:
Remove this extra line. It is what is causing `ktlint checks` to fail. Please check the [ktlint-guide](https://github.com/oppia/oppia-android/wiki/Ktlint-Guide) to avoid these mistakes in the future.

Reply:
I will follow up the guide.